package/doc/sphinx/source/conf.py

```diff
@@ -46,16 +46,8 @@ def __getattr__(cls, name):
 # documentation root, use os.path.abspath to make it absolute, like shown
 # here.
 
-# Little trick to make sure sphinx use the dev build if present
```

Comment:
Cool, so I think that's a yes to this then.

On Thu, 10 Sep 2015 06:12 kain88-de <notifications@github.com> wrote:

> In package/doc/sphinx/source/conf.py
> <https://github.com/MDAnalysis/mdanalysis/pull/423#discussion_r39125277>:
>
> > @@ -46,16 +46,8 @@ def __getattr__(cls, name):
> >  # documentation root, use os.path.abspath to make it absolute, like shown
> >  # here.
> >
> > -# Little trick to make sure sphinx use the dev build if present
>
> yes yes and double yes. Without modifying the path sphinx won't find the
> changes made on the current branch but what ever is installed globally. The
> old one is not needed anymore IMHO since it only works for python2.x, in
> Py3 they changed the build dir names. But yeah we still need the new one.
> It did cost me quite some cursing and time to figure out that path was
> wrong.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MDAnalysis/mdanalysis/pull/423/files#r39125277>.
>